src/SmartComponents/Packages/Packages.js

```diff
@@ -35,10 +38,17 @@ const Packages = () => {
     );
 
     React.useEffect(() => {
-        dispatch(fetchPackagesAction(queryParams));
+        if (firstMount) {
```

Comment:
hmm, I think we have to use this approach which is similar to the Advisories page. Details pages use the Inventory component which enables us to populate starting redux state with the decoded URL params. But yes, it would be great if we can come up with more optimal solution.